db/migrate/20190607014201_add_column_approved_to_institutions.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnApprovedToInstitutions < ActiveRecord::Migration
+  def change
+    # Note this is going to result in null values, three-state-boolean problem
```

Comment:
Regarding the [three-state-boolean](https://thoughtbot.com/blog/avoid-the-threestate-boolean-problem) problem, perhaps add in `null: false, default: false` to the `add_column` call?

Reply:
This would set all previously approved institution versions in the institution table to false which is incorrect as currently any row in the institutions table are approved due to logic that is currently in https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/models/institution_builder.rb#L86 

The plan is that after these changes and the changes in https://github.com/department-of-veterans-affairs/gibct-data-service/pull/375 are merged and deployed to production, in production a new version of institutions is created (Generate New Preview Version button below) and then published to production (Publish to production button below). 

![Screen Shot 2019-06-10 at 3 20 48 PM](https://user-images.githubusercontent.com/1094999/59435170-042d7700-8dbb-11e9-9d63-4e564eefece5.png)